pkg/plugins/golang/v2/webhook.go

```diff
@@ -63,17 +61,17 @@ validating and (or) conversion webhooks.
 }
 
 func (p *createWebhookSubcommand) BindFlags(fs *pflag.FlagSet) {
-	p.resource = &resource.Options{}
-	fs.StringVar(&p.resource.Group, "group", "", "resource Group")
-	fs.StringVar(&p.resource.Version, "version", "", "resource Version")
-	fs.StringVar(&p.resource.Kind, "kind", "", "resource Kind")
-	fs.StringVar(&p.resource.Plural, "resource", "", "resource Resource")
+	p.options = &goPlugin.Options{WebhookVersion: "v1beta1"}
```

Comment:
```suggestion
	p.options = &goPlugin.Options{WebhookVersion: "v1beta1"}
	// Since domain is used in GVK comparison, only set domain if using project version 3+.
	if p.config.IsV3() {
		p.options.Domain = p.config.Domain
	}
```